boards.txt

```diff
@@ -386,6 +386,51 @@ BBCmicrobit.menu.softdevice.s130.upload.maximum_size=151552
 BBCmicrobit.menu.softdevice.s130.build.extra_flags=-DNRF51 -DS130 -DNRF51_S130
 BBCmicrobit.menu.softdevice.s130.build.ldscript=armgcc_s130_nrf51822_xxaa.ld
 
+Calliope.name=Calliope mini
```

Comment:
What do you think about changing the key to `CalliopeMini`. If they are successful could have more than one board?